packages/web/pages/onboarding/guide.tsx

```diff
@@ -0,0 +1,96 @@
+import { useBreakpointValue } from '@metafam/ds';
+import BackgroundImage from 'assets/login-background.jpg';
+import { FullPageContainer } from 'components/Container';
+import { LandingHeader } from 'components/Landing/LandingHeader';
+import OnboardingGuidance from 'components/Landing/OnboardingGuidance';
+import { HeadComponent } from 'components/Seo';
+import { Socials } from 'pages';
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+export const getStaticProps = async () => ({
+  props: {
+    hideTopMenu: true,
+  },
+});
+
+const OnboardingGuide: React.FC = () => {
+  const responsiveBg = useBreakpointValue({
+    base: BackgroundImage,
+  });
+  const scrollContainer =
+    typeof document !== 'undefined'
+      ? document.getElementById('scroll-container')
+      : null;
+  const [section, setSection] = useState(0);
+  const hostName = useRef('https://metagame.wtf');
+  const setHostName = useCallback((host: string) => {
+    hostName.current = host;
+  }, []);
```

Comment:
Why is this a `ref`?

```suggestion
  const [hostName, setHostName] = useState('https://metagame.wtf');
```